### ISSUE TITLE
Can't Rename File within Modeling App - Gives Successful Toast

### ISSUE THREAD

**Issue by jgomez720** [KittyCAD/modeling-app#4839]
### Description
This will happen every now and then, and I haven't figured out the cause just yet, but when I go to rename a file, it says "success" but doesn't rename it.

### Version
v0.33.0

### Recordings/Screenshots
https://github.com/user-attachments/assets/74a28aa0-706a-4024-8b16-e8bd6735e24b

**Comment by jgomez720** [comment]
Sorry @farskid, I fat fingered.

**Comment by franknoirot** [comment]
Actually solid that @farskid got mentioned, we should include this in our discussion with @nadr0 on Friday because I feel it could be related to errors not being surfaced properly within fileMachine

**Comment by franknoirot** [comment]
Agreed @nadr0, I haven't dug into the root cause yet, but those are good suggested starting points.
